official/resnet/cifar10_test.py

```diff
@@ -37,14 +39,23 @@ class BaseTest(tf.test.TestCase):
   """Tests for the Cifar10 version of Resnet.
   """
 
+  _num_validation_images = None
```

Comment:
is this needed to set to None here?

Reply:
Technically, no. But my IDE complains of a variable not defined in `__init__` if I don't, and no price is too high to get rid of the squiggly underline of disapproval.